core/src/main/scala/scalacss/package.scala

```diff
@@ -61,6 +65,13 @@ package object scalacss {
   case class CssKeyframesEntry(name  : KeyframeAnimationName,
                                frames: Map[KeyframeSelector, StyleStream]) extends CssEntry
 
+  case class CssFontFace(fontFamily  : String,
+                         src         : Seq[String],
```

Comment:
Looking at it http://www.w3schools.com/cssref/css3_pr_font-face_rule.asp I assume that `src` cannot be empty.